### ISSUE TITLE
Evaluation results do not match expected results

### ISSUE THREAD

**Issue by abhirup-7** [ChunchuanLv/AMR_AS_GRAPH_PREDICTION#10]
I have trained a brand new model using all the default values and the LDC2017T10 dataset. After training and parsing the test data using generate.py, I ran the the evaluation.sh script from the amr-evaluation-tool-enhanced repository as follows:

`./evaluation.sh /home/amukherjee/AMR_AS_GRAPH_PREDICTION/Data/amr_annotation_r2/data/alignments/split/test/combined.txt_without_jamr_processed_generate /home/amukherjee/AMR_AS_GRAPH_PREDICTION/Data/amr_annotation_r2/data/alignments/split/test/combined.txt_pre_processed `

The result is:

Smatch -> P: 0.358, R: 0.343, F: 0.351
Unlabeled -> P: 0.380, R: 0.365, F: 0.373
No WSD -> P: 0.359, R: 0.344, F: 0.351
Non_sense_frames -> P: 0.038, R: 0.495, F: 0.070
Wikification -> P: 0.800, R: 0.713, F: 0.754
Named Ent. -> P: 0.836, R: 0.802, F: 0.819
Negations -> P: 0.015, R: 0.133, F: 0.026
IgnoreVars -> P: 0.121, R: 0.113, F: 0.117
Concepts -> P: 0.178, R: 0.168, F: 0.173
Frames -> P: 0.037, R: 0.489, F: 0.070
Reentrancies -> P: 0.169, R: 0.208, F: 0.187
SRL -> P: 0.259, R: 0.242, F: 0.250

This result seems completely wrong as its practically half the smatch score reported in the paper and by several people who seem to have retrained and obtained reasonably similar results. I am sure I am doing something wrong, for example using the wrong file as the gold standard to be compared against. Would appreciate some help in figuring out the problem.

Thank You!

**Comment by ChunchuanLv** [comment]
this is rather strange, I see your emailed generate_output, which seems to be completely normal.

Could you look at the generated output files and check it against the gold one ?

Chunchuan

**Comment by abhirup-7** [comment]
The problem was in the gold file. We used regex to replace all the r'~e.\d[,\d]*' expressions in the combined.txt_ file with '' as instructed by you. Thank you for your help. We have successfully reproduced your paper's results on the LDC2017T10 dataset:

Smatch -> P: 0.758, R: 0.727, F: 0.742
Unlabeled -> P: 0.786, R: 0.756, F: 0.771
No WSD -> P: 0.769, R: 0.738, F: 0.753
Non_sense_frames -> P: 0.890, R: 0.892, F: 0.891
Wikification -> P: 0.800, R: 0.713, F: 0.754
Named Ent. -> P: 0.873, R: 0.844, F: 0.858
Negations -> P: 0.639, R: 0.527, F: 0.578
IgnoreVars -> P: 0.626, R: 0.588, F: 0.606
Concepts -> P: 0.862, R: 0.854, F: 0.858
Frames -> P: 0.821, R: 0.822, F: 0.821
Reentrancies -> P: 0.521, R: 0.493, F: 0.506
SRL -> P: 0.726, R: 0.663, F: 0.693

Thank you again for taking the time out to help us figure out the issue.
